src/layers/conv.jl

```diff
@@ -393,3 +393,49 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", l.aggr)
     print(io, ")")
 end
+
+
+"""
+GINConv([fg,] nn, [eps])
+
+Graph Isomorphism Network
+
+# Arguments
+
+- `fg`: Optionally pass in a FeaturedGraph as input.
+- `nn`: A neural network/layer.
+- `eps`: Weighting factor. Default 0.
+
+"""
+
+struct GINConv{V<:AbstractFeaturedGraph,R<:Real} <: MessagePassing
+    fg::V
+    nn
+    eps::R
+end
+
+function GINConv(fg::V, nn; eps=zero(T)) where {V <: AbstractFeaturedGraph, 
+                                                T <: Real}
+    GINConv(fg, nn, eps)
+end
+
+function GINConv(nn; eps=zero(T)) where {T <: Real}
```

Comment:
I don't think you can use `zero(T)` like that. What is T supposed to be?

Change to `eps=0` or `eps=0f0`

Reply:
Done